client/src/components/MeetingExperience.tsx

```diff
@@ -73,58 +71,57 @@ export const MeetingExperience = (props: MeetingExperienceProps): JSX.Element =>
       }
     };
 
-    _createAdapters();
-  }, [credential, displayName, endpointUrl, locator, userId, onDisplayError]);
-
-  if (callWithChatAdapter) {
```

Comment:
I think we can leave this component as is and revert the changes. This `MeetingExperience` component was already converted to functional components in this PR: https://github.com/Azure-Samples/communication-services-virtual-visits-js/pull/28

Reply:
I can see that the PR is merged, but i am not able to see the changes in main 
https://github.com/Azure-Samples/communication-services-virtual-visits-sample/blob/main/client/src/components/MeetingExperience.tsx

Reply:
^ -- It is because the PR was done in the new repo (this repo) -- https://github.com/Azure-Samples/communication-services-virtual-visits-js/blob/main/client/src/components/MeetingExperience.tsx

So MeetingExperience is now a function component